fastlane/Fastfile

```diff
@@ -536,22 +536,29 @@ platform :ios do
     }
   end
 
-  desc "Export XCFramework"
-  lane :export_xcframework do |options|
+  lane :export_xcframeworks do |options|
+    create_and_sign_xcframework(scheme: 'RevenueCat', product_name: 'RevenueCat')
+    create_and_sign_xcframework(scheme: 'RevenueCatUIDev', product_name: 'RevenueCatUI')
```

Comment:
only note here: I had to use the `RevenueCatUIDev` scheme, not `RevenueCatUI` because that one doesn't archive correctly for some reason. That one is tied to SPM. I intend to remove that scheme altogether and rename `RevenueCatUIDev` -> `RevenueCatUI` to prevent confusion, but one thing at a time

Reply:
also the product name determines the actual xcframework name that gets sent out